roles/docker_portainer/tasks/main.yml

```diff
@@ -1,5 +1,10 @@
 ---
-- name: Ensure portainer data directory exist
+- name: Set portainer volume directory for NFS usage
```

Comment:
Interesting, but I think ultimately unnecessary. There is no code where this variable is used oin both use cases (nfs / no nfs).
Currently used in the "Ensure portainer data folder exist" task but this cannot work with nfs.